docs/ref/settings.txt

```diff
@@ -665,6 +665,8 @@ as the general :setting:`TIME_ZONE` setting.
 
 When :setting:`USE_TZ` is ``True`` and this option is set, reading datetimes
 from the database returns aware datetimes in this time zone instead of UTC.
+When this option is ``None`` then UTC will be used.
```

Comment:
I value the clarification but I still have questions (I think I now the answers, but I fear other readers may be confused):

1. UTC will be used and the returned timezones will be aware or not?
2. `this option` refers to `USE_TZ` or `TIME_ZONE`?

So I would suggest instead:

```suggestion
When :setting:`USE_TZ` is ``True``, reading datetimes from the database
returns aware datetimes.  If this option is set, the returned datetimes will
be in this time zone, otherwise UTC will be used.
```

Reply:
1. Yep with `USE_TZ` then returned timestamps will be aware 
2. `TIME_ZONE`

The suggestion is ok but I'm not sure I think it's "better" 🤔 because it brings the "aware" part to the front which is secondary.

I guess the challenge here is explaining this pseudocode in natural language:

```
if USE_TZ = False:
   error!

else:
   timezone = DATABASES.TIME_ZONE
   if timezone is None:
        timezone = 'UTC'
```

The fact that timestamps are "aware" is secondary/implied if that makes sense? 🤔 

If you think that the suggestion makes that ^ pseudocode clear then I'm ok.

Reply:
Thank you! I gotta confess, as I web service developer, I think I first try to confirm whether timestamps will be aware or not, and then, if "yes", I seek information about what timezones they will be in. But, OTOH, the first question should be answered (or seek answers in) the details of the `USE_TZ` setting, so I can see where you are coming from :-).

How about:

```suggestion
When :setting:`USE_TZ` is ``True``, reading datetimes from the database
returns aware datetimes with the timezone set to the value specified in
this option if not ``None``, or to ``UTC`` otherwise.
```